science/hacktv/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           github 1.0
+
+github.setup        fsphil hacktv b3d9ecb87f042bb6729824fabb40d55d145c3e65
+version             0.1.20221001
+
+categories          science
+license             GPL-3+
+maintainers         {@kesterlester cern.ch:lester} \
+                    openmaintainer
+description         Broadcast TV using an SDR like HackRF One
+long_description    Broadcast TV using a Software Defined Radio like HackRF One
+
+checksums           sha256  9d2f65aa25427983192331c3cf0b1e88702c0392e02dec5184bb8efcb5657563 \
+                    rmd160  6a64cf2f5fe7f4b2ab1fe492c9441b31327136bd \
+                    size    255617
+
+patchfiles          destdir-fix.diff
+
+depends_build       port:pkgconfig
+
+depends_lib         port:ffmpeg-devel \
+                    port:SoapySDR \
+                    port:hackrf
+build.args-append       MACPORTS_CC=${configure.cc} \
+                        CXX=${configure.cxx} \
+                        CPP=${configure.cpp}
```

Comment:
You may avoid this line by adding:
```tcl
PortGroup        makefile 1.0
```

Reply:
Am guessing that would not add my lines 

```
build.args-append       MACPORTS_CC=${configure.cc} \
                        CXX=${configure.cxx} \
                        CPP=${configure.cpp}
```

but would instead add something like

```
build.args-append       CC=${configure.cc} \
                        CXX=${configure.cxx} \
                        CPP=${configure.cpp}
```
meaning that I'd then need to change the makefile diff, which is a more complex thing to maintain than this.

Look, I don't know. 

This is not really my thing. I should be writing a lecture for some undergrads I have to lecture to tomorrow morning. I'm not really that into code golf.  Am very happy to fix real weaknesses, security holes, supremely non-portable statements.  But when it's just about aesthetics I'm running out of motivation to carry on.

Reply:
@kesterlester if you move forward with my suggestion to patch Makefile when you may remove it ;)

Reply:
OK - I saw that now.  I think I understand what's going on.

But one thing I don't understand is that this merge request has suddenly gone "closed".

I haven't yet managed figure out whether that means it was rejected, or whether it means it was accepted, or whether it means I have to do something else.

I may be able to find out myself, but do let me know if I should worry.

Reply:
Not worry, you're quite close :)

The only things that I see is missed is `CFLAGS`, and I suggest to patch Makefile to enjoy a PortGroup which takes care of it.

Reply:
I think I did it. But I really have to do something else now. Working week ahead.